Add information about the Meilisearch 1.2.0 upgrade (#1220)
Enalean/tuleap-documentation-en: languages/en/deployment-guide/14.x.rst

  Tuleap 14.11 is currently under development.

Upgrade of the local Meilisearch instance from 1.0 to 1.2
---------------------------------------------------------

> If you have you have deployed a :ref:`local Meilisearch instance <fts-local-meilisearch>` it will be automatically

Reviewer: ```suggestion
If you have deployed a :ref:`local Meilisearch instance <fts-local-meilisearch>`, it will be automatically
```

upgraded to the 1.2.0 version. The process is automatic when the server is restarted after its update.

> The upgrade will re-index all the items. Some items might not been found in a search and an increased load

Reviewer: ```suggestion
The upgrade will re-index all the items. Until the process is completed, some items might not be found when searching, and an increased load on your instance is expected.
```

on your instance is expected until the process is completed.

> Administrators having deployed an independant Meilisearch instance should also

Reviewer: ```suggestion
Administrators having deployed an independent Meilisearch instance should also
```
